packages/iobio-react-components/src/stats_util.mts

```diff
@@ -0,0 +1,64 @@
+/*
```

Comment:
Main Stats utility

follow up PR to start adding folder organization separating FE / BE concerns:
https://github.com/overture-stack/iobio-components/pull/45